web-apps.txt

```diff
@@ -1,7 +0,0 @@
-duolingo
```

Comment:
Why's this file removed here? It is still used by `desktop_object.py`. Not that I expect that script ever to be run again…

Reply:
The file is not removed, it is empty now with these changes as there is no other "webapp" left - I didn't remove `desktop_object.py` as I wasn't sure whether this was used somewhere else but can remove it also if you prefer.